src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp

```diff
@@ -230,8 +230,6 @@ bool medVtkViewItkDataImageInteractor::SetViewInput(medAbstractData* data, int l
         bRes = false;
     }
 
-    delete poOldConv;
```

Comment:
Line 208 replaces `m_poConv` with a new instance, so removing this line will cause a memory leak as the old instance is no longer deleted.